Anki/public/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>Anki</title>
+    <link rel="stylesheet" href="assets\css\main.css" />
+    <script  src="assets\js\font_awesome.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/8.3.1/firebase-app.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/8.3.1/firebase-analytics.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/8.3.1/firebase-auth.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/8.3.1/firebase-firestore.js"></script>
+    <script src="assets\js\auth.js"></script>
+</head>
+<body>
+    <header id="menu">
+        <input type="checkbox" id="menuCheckbox" class="menu-checkbox">
+        <label for="menuCheckbox" class="menu-icon">
+            <div id ="menuIconLine1" class="menu-icon_line"></div>
+            <div id ="menuIconLine2" class="menu-icon_line"></div>
+            <div id ="menuIconLine3" class="menu-icon_line"></div>
+        </label>
+        <nav id="menuSection1" class="navbar">
+            <a class="navbar__link_logo" href="/#/home">Anki</a>
+            <a class="navbar__link" href="/#/home">Home</a>
+            <a class="navbar__link" href="/#/collections">Collections</a>
+            <a class="navbar__link" href="/#/cards">Cards</a>
+            <a class="navbar__link" href="/#/statistics">Statistics</a>
+            <a class="navbar__link" href="/#/privacy">Privacy</a>
+        </nav>
+        <div id="menuSection2" class="header-section">
+            <button id="loginButton" onclick="window.location='/#/login'" class="button-animated button-secondary">Log in</button>
+            <button id="registerButton" onclick="window.location='/#/register'" class="button-animated button-secondary">Register</button>
+            <button id="userButton" onclick="window.location='/#/user'" class="button-animated button-secondary">User</button>
+            <button onclick="logout()" id="logoutButton" class="button-animated button-secondary">Log out</button>
+        </div>
+    </header>
+    <main id="root">
+    </main>
+    <footer>
+        <div class="container">2021-Anki-<a href="/#/privacy">Privacy</a></div>
+        <div class="container">
+            <label class="theme-switch" for="themeCheckbox">
+                <input type="checkbox" id="themeCheckbox"/>
```

Comment:
When the checkbox is completely hidden via `display: none` it is not accessible via keyboard by users. Try something else instead.

Reply:
I changed `display: none` to `opacity: 0` and when the check-box has focus the slider has a box-shadow.
The same problem was with the checkbox for the menu.